example.flapjack.ini

```diff
@@ -76,12 +76,16 @@ add_extensions = org.gnome.Sdk.Docs//3.26:share/runtime/docs
 # This is the list of modules that you want to be able to open and hack on.
 
 modules =
-    xapian-glib
     eos-metrics
```

Comment:
Seems unrelated?

Reply:
I just took all the editable modules in the order they were in the manifest, so I think this change is good, but I'll update the commit message to explain.

Reply:
Okay makes sense